rhoas-client/src/main/resources/application.properties

```diff
@@ -0,0 +1,21 @@
+event-bridge.rhoas.mgmt-api.host=https://api.openshift.com
+event-bridge.rhoas.instance-api.host=http://localhost
+event-bridge.rhoas.sso.red-hat.auth-server-url=https://sso.redhat.com/auth/realms/redhat-external
+event-bridge.rhoas.sso.red-hat.refresh-token=dummy-refresh-token
```

Comment:
What does `dummy-refresh-token` mean?

Reply:
It's just a placeholder to make it clear where to put the refresh token for Red Hat SSO (aka offline token in this case)

Reply:
@kostola Can you make this accept an ENV variable as that is how we will configure the offline token in our deployment

Reply:
@robpblake Quarkus already translates env variables to properties if formatted correctly. I put all the property / env variables pair in the README file now, removing the default values.